apps/e2e/cypress/e2e/genericTemplates.cy.ts

```diff
@@ -726,6 +726,43 @@ context('GenericTemplates tests', () => {
 
       cy.contains(genericTemplateTitle);
     });
+
+    it('should be a character limit to the template proposal question for office user', () => {
+      cy.createTemplate({
+        name: proposalTemplateName,
+        groupId: TemplateGroupId.PROPOSAL,
+      });
+      const char256 =
```

Comment:
Instead of hardcoding this you can maybe use `faker.string.alphanumeric(256)` to generate random 256 characters long string.

Reply:
that sounds better than what i have going on, ill add that change

Reply:
![image](https://github.com/UserOfficeProject/user-office-core/assets/144329813/dff4854b-1558-495e-8e03-672926bb2c4d)

Reply:
This is what I found on their documentation site https://fakerjs.dev/api/string.html#sample